auto_round/export/export_to_itrex/export.py

```diff
@@ -185,15 +185,11 @@ def pack_model(
             5. zeros is always needed even for sym.
         inplace (bool, optional): Compress the model in place, or copy the model and compress it.
 
-    xpu args:
-        compression_dtype=torch.int8,
-        compression_dim=0,
-        use_optimum_format=False,
-        scale_dtype=convert_dtype_str2torch(config.scale_dtype),
-        device="xpu",
     """
+    # Due to XPU doesn't support tuning yet
+    device = detect_device() if device == "xpu" else device
     if model.device.type == 'meta':
-        model = model.to("cuda" if device == "xpu" else device)
+        model = model.to(device)
```

Comment:
could xpu device run the exporting successfully?

Reply:
It is actually running on the CPU, so it will logically succeed. However, the speed depends on the machine's CPU efficiency.